source/client-side-encryption/tests/README.rst

```diff
@@ -67,6 +67,8 @@ The following matches a command_started_event for an insert of a document where
 
 The values of `$$type` correspond to `these documented string representations of BSON types <https://www.mongodb.com/docs/manual/reference/bson-types/>`_.
 
+- Addition of `$$exists`. The semantics match the `Unified Test Format <https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#exists>`_.
```

Comment:
Can we do without this change? The [Transactions test format](https://github.com/mongodb/specifications/tree/master/source/transactions/tests#null-values) already supports this:

> Tests MUST assert that the actual command omits any field that has a `null` value in the expected command.

Side note: seeing this made me consider how this PR is effectively piling on technical debt for migrating CSFLE tests to the unified format ([DRIVERS-2295](https://jira.mongodb.org/browse/DRIVERS-2295)), assuming we ever get to that.

Reply:
Nice. Reverted addition of `$$exists` and replaced assertions with `null`.